src/app/components/elements/CareersBanner.tsx

```diff
@@ -30,11 +30,11 @@ interface CareerCardProps {
 }
 
 const CareerCard = ({ imgSrc, imgAlt, title, text, linkTo, linkText, children }: CareerCardProps) => (
-  <Card className="career-card h-100">
+  <Card className="career-card h-100 w-100">
     {imgSrc && <CardImg variant="top" src={imgSrc} alt={imgAlt} className="career-media-row-image" />}
     <CardBody className="career-card-body d-flex flex-wrap flex-column">
       {children}
-      <CardTitle className="career-subtitle">{title}</CardTitle>
```

Comment:
i don't agree with this change. what was it done to fix? to make it look like the news card? 

If so, i'd look to change the news card - as this is using the bootstrap styling which is preferable :)